provisioning/inventory/hosts

```diff
@@ -1,6 +1,6 @@
 [devservers]
 # TODO: relative path to private key breaks ability to run vagrant up from anywhere in the project
-vagrant ansible_ssh_host=127.0.0.1 ansible_ssh_port=2222 ansible_ssh_user=vagrant ansible_ssh_private_key_file=./.vagrant/machines/default/virtualbox/private_key
+vagrant ansible_ssh_host=127.0.0.1 ansible_ssh_port=2222 ansible_ssh_user=vagrant
```

Comment:
How is ansible being able to auth with no key or password? Are you positive this works ok?